### PR TITLE
new games code added

### PR DESCRIPTION
**Pull request by walkwel**

### CODE REVIEW

File: src/components/games/passengerPickup/src/component.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
```
Comment (FeynmanDNA) [inline src/components/games/passengerPickup/src/component.js]: are we still disabling the eslint?

File: src/components/games/passengerPickup/src/index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
+<script src="./code/control.js"></script>
+<script src="./code/level1.js"></script>
+<script src="./code/level3.js"></script>
+<script src="./code/simulation.js"></script>
+<p id="score1"></p>
```
Comment (FeynmanDNA) [inline src/components/games/passengerPickup/src/index.html]: indent the code

File: src/components/games/passengerPickup/src/index.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    ctx1.fillStyle = "#00FF00";
+    ctx1.fillRect(bot1_2.x, bot1_2.y, botSize, botSize);
+
+    ctx2.fillStyle = "#0000FF";
+    ctx2.fillRect(bot2_1.x, bot2_1.y, botSize, botSize);
+
+    ctx2.fillStyle = "#000000";
+    ctx2.fillRect(bot2_2.x, bot2_2.y, botSize, botSize);
+  }, 10);
+</script>
```
Comment (FeynmanDNA) [inline src/components/games/passengerPickup/src/index.html]: is this for CodeSandbox? do we need to have a index.html within achievement?

File: src/components/games/passengerPickup/public/myEditor.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+
+import sys
+import time
+import traceback
+import javascript
+
+from browser import document as doc, window, alert, console
+
+def createFunctionFromPython(string):
+	try:
```
Comment (FeynmanDNA) [inline src/components/games/passengerPickup/public/myEditor.py]: indent the py code with 4 spaces

File: src/components/games/passengerPickup/src/index.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        <ol>
+          {
+            Object.keys(playerKeys).map(key => <li key={key}> {key.toUpperCase()}: {playerKeys[key]} </li>)
+          }
+        </ol>
+      </div>}
+      <p>URL params used to customize game are</p>
+      <ol>
+        <li>mode : 'manual' || 'custom'</li>
+        <li>level : 1 || 2 || 3 [max 3]</li>
```
Comment (FeynmanDNA) [inline src/components/games/passengerPickup/src/index.js]: do not use || in HTML, this is confusing in the JSX

File: src/components/games/passengerPickup/src/simulation/level1.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
+/* eslint-disable */
+var bot = function(world) {
+	var findShortestPath = function(arr, pointA, pointB, charId){
```
Comment (FeynmanDNA) [inline src/components/games/passengerPickup/src/simulation/level1.js]: eslint this code with better indentation

**Comment by walkwel** [comment]
I will make all these changes in the Games repo and then include the code here.

**Comment by FeynmanDNA** [comment]
thanks

**Pull request closed without merging**